bson/bsoncodec/struct_codec.go

```diff
@@ -393,7 +393,7 @@ func (sc *StructCodec) DecodeValue(dc DecodeContext, vr bsonrw.ValueReader, val
 
 func isZero(v reflect.Value, omitZeroStruct bool) bool {
 	kind := v.Kind()
-	if (kind != reflect.Pointer || !v.IsNil()) && v.Type().Implements(tZeroer) {
+	if (kind != reflect.Ptr || !v.IsNil()) && v.Type().Implements(tZeroer) {
```

Comment:
nit, `Ptr` is the old name of `Pointer`

From [reflect/type.go](https://github.com/golang/go/blob/master/src/reflect/type.go#L280-L281):
```go
// Ptr is the old name for the Pointer kind.
const Ptr = Pointer
```

Reply:
You're correct. However, the Go driver currently supports a min Go version of 1.13, which doesn't have the new `Pointer` symbol, so we have to use the old `Ptr` symbol.